packages/sit-onyx/src/components/OnyxTooltip/OnyxTooltip.vue

```diff
@@ -99,10 +112,10 @@ watch(type, () => (ariaPattern.value = createPattern()));
 const tooltip = computed(() => ariaPattern.value?.elements.tooltip);
 const trigger = computed(() => toValue<object>(ariaPattern.value?.elements.trigger));
 
-const tooltipWrapperRef = ref<HTMLElement>();
-const tooltipRef = ref<HTMLElement>();
-const { openDirection, updateOpenDirection } = useOpenDirection(tooltipWrapperRef, "top");
-const { wedgePosition, updateWedgePosition } = useWedgePosition(tooltipWrapperRef, tooltipRef);
+const tooltipWrapper = useTemplateRef("tooltipWrapperRef");
+const tooltipRef = useTemplateRef("tooltipRef");
```

Comment:
We should rename the ref or the variable so they are not the same because of the bug #2557